src/main/java/Controller/playAudioController.java

```diff
@@ -0,0 +1,29 @@
+package Controller;
+
+import InputBoundary.playAudioInputBoundary;
+import InputData.audioInputData;
+
+/**
+ * Add Java docs after
+ */
+public class playAudioController {
+
+    private final playAudioInputBoundary playAudioInteractor;
+    private final audioInputData songFile;
+
+    /**
+     * Add Java docs after
+     */
+    public playAudioController(playAudioInputBoundary playAudioInteractor, audioInputData songFile) {
+        this.playAudioInteractor = playAudioInteractor;
+        this.songFile = songFile;
```

Comment:
I might be wrong, but I was under the impression that the controller _constructs_ the audioInputData as opposed to simply being passed the already-existing audioInputData (the alternative would be the viewModel constructing the audioInputData, which IIRC we discussed in office hours and was incorrect/violates clean architecture). Could someone confirm? (I bring this up because I was wondering about how to approach this in my own use case.)